### PR TITLE
Codorials

### PR DESCRIPTION
**Pull request by Ehesp**

### CODE REVIEW

**Comment by Ehesp** [comment]
cc @chrisbianca @Salakar 

https://codorial.com/g/invertase/react-native-firebase/authentication-with-firebase/08?branch=codorials

**Comment by Salakar** [comment]
@Ehesp is the auth tutorial finished? Feels like it ends abruptly on the Facebook auth?

**Comment by Ehesp** [comment]
Yeah it's finished - I can add a last page with a summary?

File: codorials/authentication-with-firebase/facebook-login.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+      if (!data) {
+        throw new Error('Something went wrong obtaining the users access token'); // Handle this however fits the flow of your app
+      }
+
+      // create a new firebase credential with the token
+      const credential = firebase.auth.FacebookAuthProvider.credential(data.accessToken);
+
+      // login with credential
+      await firebase.auth().signInWithCredential(credential);
```
Comment (Salakar) [inline codorials/authentication-with-firebase/facebook-login.md]: Should be signInAndRetrieveDataWithCredential

**Comment by Salakar** [comment]
I think a summary would close it off nicely - also could maybe do with bringing a quick version of [this](https://rnfirebase.io/docs/v3.3.x/auth/social-auth#Google) as a page for completeness - but should mention to use our fork of it I guess @chrisbianca ?: https://github.com/invertase/react-native-google-signin - can't remember what the issue was with their repo

File: codorials/authentication-with-firebase/understanding-firebase-auth.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+
+```js
+import firebase from 'react-native-firebase';
+
+firebase.auth().createUserAndRetrieveDataWithEmailAndPassword('jim.bob@gmail.com', 'supersecret!')
+  .then((user) => {
+    console.log('New User', user);
+  })
+  .catch((error) => {
+    console.error('Woops, something went wrong!, error);
```
Comment (Salakar) [inline codorials/authentication-with-firebase/understanding-firebase-auth.md]: Missing a single quote after the exclamation mark

**Comment by Ehesp** [comment]
Added in two new pages; Google Login & Conclusion.

**Pull request merged**